test/e2e/postinstall/syncsets/syncsets_suite_test.go

```diff
@@ -125,7 +126,10 @@ var _ = Describe("Test Syncset and SelectorSyncSet func", func() {
 				resultConfigMap = &corev1.ConfigMap{}
 				err = targetClusterClient.Get(ctx, client.ObjectKey{Name: "foo", Namespace: "default"}, resultConfigMap)
 				Ω(err).ShouldNot(HaveOccurred())
-				Ω(resultConfigMap.Data).Should(Equal(map[string]string{"foo": "bar"}))
+				Ω(resultConfigMap.Data["foo"]).Should(Equal("bar"))
```

Comment:
Why change this? Don't we want to ensure that the configmap synced has only the one entry? With this change, the test would still pass if there were other entries besides the "foo" entry.

Reply:
It feels overly fragile with too high a cost to uncover. If something were to start adding annotations then this would only fail after an extremely time consuming e2e run. It doesn't seem like a great idea to assume we're the only thing possibly setting annotations even if that's the case for the most part. I'd rather we test what we know we want to be there.

Reply:
This is not checking the annotations, though. This is checking the `data` field.

Reply:
I am just straight up confused, my bad, will address.